tests/functional/simple_snapshot/test_hard_delete_snapshot.py

```diff
@@ -14,6 +18,14 @@
 # These tests uses the same seed data, containing 20 records of which we hard delete the last 10.
 # These deleted records set the dbt_valid_to to time the snapshot was ran.
 
+# Using replace on a timestamp won't account for hour differences unless given the local timezone.
+# We can force python as utc but not postgres fields which need to be handled as local timestamps.
+def currenttz():
+    if time.daylight:
+        return timezone(timedelta(seconds=-time.altzone), time.tzname[1])
+    else:
+        return timezone(timedelta(seconds=-time.timezone), time.tzname[0])
```

Comment:
I suspect there might be a better way to do this, but I'm find merging this if it's not obvious.

Reply:
Honestly, it's hard to find an answer in the std lib. I went with this to avoid adding a library. Not tied to any which way though.